x-pack/qa/sql/src/main/java/org/elasticsearch/xpack/qa/sql/jdbc/DataLoader.java

```diff
@@ -32,26 +32,37 @@
 
     public static void main(String[] args) throws Exception {
         try (RestClient client = RestClient.builder(new HttpHost("localhost", 9200)).build()) {
-            loadDatasetIntoEs(client);
+            loadEmpDatasetIntoEs(client);
             Loggers.getLogger(DataLoader.class).info("Data loaded");
         }
     }
 
     protected static void loadDatasetIntoEs(RestClient client) throws Exception {
-        loadDatasetIntoEs(client, "test_emp");
-        loadDatasetIntoEs(client, "test_emp_copy");
+        loadEmpDatasetIntoEs(client);
+    }
+
+    protected static void loadEmpDatasetIntoEs(RestClient client) throws Exception {
+        loadEmpDatasetIntoEs(client, "test_emp");
+        loadEmpDatasetIntoEs(client, "test_emp_copy");
         makeAlias(client, "test_alias", "test_emp", "test_emp_copy");
         makeAlias(client, "test_alias_emp", "test_emp", "test_emp_copy");
     }
 
+    protected static void loadDocsDatasetIntoEs(RestClient client) throws Exception {
+        loadEmpDatasetIntoEs(client, "emp");
```

Comment:
I'd prefer to rename the one in tests to `emp` and load the `library` one for all the tests.